notify_listener.go

```diff
@@ -267,11 +275,16 @@ func (l *NotifyListener) processChangeset(event *store.Event) {
 		}
 
 		for k, v := range newValues {
-			// Maps are not supported. They can break checksum validation
-			// when re-marshaling. Pass the original JSON string instead.
-			switch v.(type) {
-			case map[string]interface{}:
-				v = string(newRawValues[k])
+			columnName := k
+			columnType, ok := columnTypes[fullTableName][columnName]
+			if !ok {
+				panic("column type could not be determined")
```

Comment:
Ah. Might want to change these. See what you think?

Reply:
I'm fine with this temporarily, but this should never happen during "runtime" just during pre-checks.

Reply:
It "should" never happen at all. It would have to be an internal problem.